gym_compete/policy.py

```diff
@@ -212,19 +213,20 @@ def loop_fn(time, cell_output, cell_state, loop_state):
                     last_out = seq_to_batch(last_out)
                     self.state_out.append(final_state)
 
-                    return last_out
+                    return last_out, input_seq
 
-                value_out = lstm(0, 'v')
+                value_out, value_ff_acts = lstm(0, 'v')
```

Comment:
I think we're unlikely to want to use value activations since they don't directly determine the action taken. But no harm in exposing it.